token/cli/src/config.rs

```diff
@@ -276,42 +322,52 @@ impl<'a> Config<'a> {
         override_name: &str,
         wallet_manager: &mut Option<Arc<RemoteWalletManager>>,
         token: Option<Pubkey>,
-    ) -> Pubkey {
-        if let Some(address) = pubkey_of_signer(arg_matches, override_name, wallet_manager).unwrap()
+    ) -> Result<Pubkey, Error> {
+        if let Some(address) = pubkey_of_signer(arg_matches, override_name, wallet_manager).ok().flatten()
```

Comment:
I was the one being dense.  For some reason I was thinking that unwrap() would panic if it was Err.  I will revert this.

Reply:
I looked into a bit more, and this will swallow the error, which means that if someone has provided the `"address"` parameter, and there's nothing at that path, there won't be an error, which would be bad.  I think we should keep the old behavior